bridge/client/app/_components/ktb-create-secret-form/ktb-create-secret-form.component.spec.ts

```diff
@@ -1,27 +1,34 @@
-import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { ComponentFixture, fakeAsync, TestBed, waitForAsync } from '@angular/core/testing';
 
 import { KtbCreateSecretFormComponent } from './ktb-create-secret-form.component';
-import {AppModule} from '../../app.module';
-import {HttpClientTestingModule} from '@angular/common/http/testing';
+import { AppModule } from '../../app.module';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
 
-describe('KtbUniformViewComponent', () => {
+describe('KtbCreateSecretFormComponent', () => {
   let component: KtbCreateSecretFormComponent;
   let fixture: ComponentFixture<KtbCreateSecretFormComponent>;
 
   beforeEach(waitForAsync(() => {
     TestBed.configureTestingModule({
-      declarations: [ KtbCreateSecretFormComponent ],
       imports: [
         AppModule,
-        HttpClientTestingModule,
-      ]
+        HttpClientTestingModule
+      ],
```

Comment:
Let's add the trailing comma here? Because on other places we also have it.
```suggestion
        HttpClientTestingModule,
      ],
```